### PR TITLE
Syntax

### PR DESCRIPTION
**Pull request by jutsuman**
The README ist ready and the synthax is OK.

### CODE REVIEW

**Comment by jutsuman** [comment]
It's OK yet.

**Comment by logroot** [comment]
What hve you done !?### 

1. > 

- [ ]   @pablito234  `_****_` :-1:

**Comment by jutsuman** [comment]
No it's ok guy

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+- Es ist ein C Programm, also gcc muss in Ihrem Computer vorhanden sein.
+- Die Datei main.c mit gcc kompilieren lassen, um die Execute-Datei zu generieren.
+- Die Execute-Datei ist auch dabei falls nötig.
+
+# Entwicklung:
+
+Gerne dürfen Sie das Programm entwickeln. Dafür klonen Sie das Verzeichnis in Ihrem Computer.
+Wenn Sie nicht wissen wie man ein Verzeichnis in Github klont, dann müssen Sie zuerst lernen wie man Github bzw. Git nutzt.
+
+# Dankbarkeit:
```
Comment (logroot) [inline README.md:20]: nice hey

**Review by logroot — COMMENTED** [review]
nice job @jutsuman 
:+1:

File: README.md (hunk 1 above)
Comment (jutsuman) [reply README.md:20]: thanks

**Comment by logroot** [comment]
oki

**Review by logroot — APPROVED** [review]
:100:

**Pull request merged**